cli/flox/doc/flox.md

```diff
@@ -1,76 +1,33 @@
-% FLOX(1) Flox User Manuals
+---
+title: FLOX
+section: 1
+header: "Flox User Manuals"
+...
 
 # NAME
 
 flox - command-line interface (CLI)
```

Comment:
```suggestion
flox - Developer environments you can take with you
```

Reply:
Fixed (but lower case to match all the other man pages)